packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -186,6 +188,33 @@ If a misconfigured constraint is added, `synth` will fail with an error message.
 
 Read more at [Service Catalog Constraints](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/constraints.html).
 
+### Deploy with StackSets
+
+A StackSets deployment constraint allows you to configure product deployment options using 
+[AWS CloudFormation StackSets](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/using-stacksets.html). 
+You can specify multiple accounts and regions for the product launch following StackSets conventions.
+There is an additional field `allowStackControl` that configures ability for end users to create, edit, or delete the stacks. 
+End users can manage those accounts and determine where products deploy and the order of deployment.
+You can only define one StackSets deployment configuration per portfolio-product association,
+and you cannot both set a launch role and StackSets deployment configuration for an assocation.
+
+```ts fixture=portfolio-product
+const accounts = ['012345678901', '012345678902', '012345678903'];
+
+const regions = ['us-west-1', 'us-east-1', 'us-west-2', 'us-east-1'];
+
+const adminRole = new iam.Role(this, 'AdminRole', {
+  assumedBy: new iam.AccountRootPrincipal(),
+});
+
+const executionRole = new iam.Role(this, 'ExecutionRole', {
+  assumedBy: new iam.AccountRootPrincipal(),
+});
+
+portfolio.deployWithStackSets(product, accounts, regions, adminRole, executionRole, true);
```

Comment:
this was more just for simplicity.  The execution role does need to be in the child account but creating it here for examples seemed out of scope, since it might be an imported role or defined elsewhere since its deploying to different accounts.   
If this example makes it too confusing, I will try to get better minimal end user example.

Reply:
Let's edit the example. The examples in the ReadMe should be realistic.

Reply:
got it.

Reply:
Not sure if updated example is really good, but the execution role is created and done in a different stack/pipeline since its in the target accounts. I think importing makes some sense.

Reply:
One redesign might be that the actual L1 just takes a RoleName.  Since CDK likes object references and not just attributes it felt normal to have it accept role and disambiguate that on the backend, since the users should hopefully also defining their roles via cdk in some other stack.  But we could just go with `executionRoleName: string`.

Reply:
No. We should keep them as `IRole`.